test/Templates.Test/Helpers/TemplateTestBase.cs

```diff
@@ -42,18 +42,14 @@ public TemplateTestBase(ITestOutputHelper output)
             // Directory.Build.props/targets context
 
             var templatesTestsPropsFilePath = Path.Combine(basePath, "TemplateTests.props");
-            var templatesTestsTargetsFilePath = Path.Combine(basePath, "TemplateTests.targets");
             var directoryBuildPropsContent =
 $@"<Project>
     <Import Project=""{templatesTestsPropsFilePath}"" />
     <Import Project=""Directory.Build.After.props"" Condition=""Exists('Directory.Build.After.props')"" />
 </Project>";
             File.WriteAllText(Path.Combine(TemplateOutputDir, "Directory.Build.props"), directoryBuildPropsContent);
 
-            var directoryBuildTargetsContent =
-$@"<Project>
-    <Import Project=""{templatesTestsTargetsFilePath}"" />
-</Project>";
+            const string directoryBuildTargetsContent = "<Project></Project>";
```

Comment:
nit: https://github.com/aspnet/Templating/blob/51461991c891de628b7489425fb217a5a968390f/test/Templates.Test/Helpers/TemplateTestBase.cs#L52 would be sufficient here.